deployment/helm/values.yaml

```diff
@@ -88,7 +101,7 @@ debugEcho:
   - "foo1"
   # shared defines whether instances created as a result of
   # applying this debugEcho configuration as shared
-  shared: true
+  shared: false
```

Comment:
Why is this included?

Reply:
Good catch. This was an accident. Will remove. Though i do think that `shared` in DebugEcho should default to `false`.